pydis_site/README.md

```diff
@@ -0,0 +1,52 @@
+# `pydis_site` project directory
+
+This directory hosts the root of our **Django project**[^1], and is responsible
+for all logic powering our website. Let's go over the directories in detail:
+
+- [`apps`](./apps) contains our **Django apps**. If you want to add your own API
```

Comment:
Is this written assuming some prior knowledge of Django e.g. what an "app" means in the Django world? Otherwise a glossary footnote would be nicer, like you did for the project.

Reply:
> `apps` contain our Django apps, which are the building blocks that make up our Django project. A Django project must always consist of one or more apps, and these apps can be made completely modular and reusable across any Django project. For more information, see the official Django tutorial, [which has a section on reusable apps](https://docs.djangoproject.com/en/4.0/intro/reusable-apps/).

Reply:
Thanks, that's a good one. See fee12689